README.md

```diff
@@ -297,13 +297,20 @@ folder in the repository.
 
 ### Vercel
 
-```sh
-vercel
+Add `.vercel.json` in the project root directory with the following:
+
+```json
+{
+  "buildCommand": "npm run build --with-vercel",
+  "devCommand": "npm run start"
+}
```

Comment:
Nice. Does it work?
We can omit `--with-vercel` as it detects from an env var.
I'm not sure how `devCommand` is used.
Don't we need "output directory"?

Reply:
I'm still figuring it out. I think the best solution is vercel add waku commands into their document

Reply:
Yeah, we can send them a PR. But, I hope to have a meantime solution too.